README.md

```diff
@@ -1,12 +1,21 @@
-#This project was abandoned because [Moxie Marlinspike said he was not OK with LibreSignal using the Open Whisper Systems servers and the name "Signal"](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165).
+#This GitHub repository has been abandoned.
```

Comment:
@xmikos Thanks!  I apologize for getting that the other way around. The timeline for when LibreSignal started was a bit blurry. I have now dug around a bit. Correct me if anything looks incorrect:

* In September 2014, @JavaJens attempted [a pull request](https://github.com/WhisperSystems/Signal-Android/pull/1960) to add websocket support to TextSecure. This was reviewed by Moxie, who [commented that](https://github.com/WhisperSystems/Signal-Android/pull/1960#issuecomment-63234532) the code was not in very good shape and was cut and pasted from third party libraries. JavaJens closed the PR and moved everything to [his own GitHub repo](https://github.com/JavaJens/TextSecure).
* In August 2015, xmikos was already distributing an independent build of TextSecure, but then also [started distributing](https://twitter.com/xmikos/status/636494031464321024) a build that was based on JavaJens's websocket repo, which he initially also called TextSecure. Moxie asked xmikos to rebrand his builds, so xmikos renamed both his independent build and his JavaJens-websocket-build as TextLibre.
* In November 2015, TextSecure and RedPhone merged to become Signal, so xmikos renamed both his independent build and his JavaJens-websocket-build as as LibreSignal.
* In February 2016, mimi89999 [decided](https://github.com/JavaJens/TextSecure/issues/72) to fork JavaJens's GitHub repo to create this LibreSignal GitHub repo, and Xmikos started basing his websocket-build on this repo.